api/Worker.json

```diff
@@ -23,6 +23,15 @@
             "version_added": "10"
           },
           "nodejs": [
+            {
+              "version_added": "12.17.0",
```

Comment:
CI is failing on this line because we have not yet recorded 12.17 as a valid release. To make this PR pass the tests, you'll need to add 12.17 to [`nodejs.json`](https://github.com/mdn/browser-compat-data/blob/master/browsers/nodejs.json).